RELEASE-NOTES.txt

```diff
@@ -1,6 +1,6 @@
 2.16
 -----
-
+- Updated the account deletion flow for clarity #1026
```

Comment:
@charliescheer It looks like this change was made as part of #1026, but that PR was merged to `release/2.15`, so it's being shipped with this version. Are you sure you want to include the release note for that as part of `2.16`? I think that might be a little confusing - for us.

Unfortunately we can't include it as part of `2.15`, because it's not translated.

Reply:
That was supposed to be in 2.15  I must have added it to the wrong tag.  Sorry about that.  Since it can't be included in 2.15, I will fix this up in a PR off of develop, so it will be correct with the next release when ever 2.16 happens